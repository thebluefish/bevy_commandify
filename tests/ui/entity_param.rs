use bevy_commandify::*;

#[entity_command]
fn foo(world: &mut World) { }

/// Test that an entity_command requires an `Entity` fn parameter
fn main() { }