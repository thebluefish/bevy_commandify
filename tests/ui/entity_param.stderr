error: Entity commands must take in a `Entity` parameter
 --> tests/ui/entity_param.rs:3:1
  |
3 | #[entity_command]
  | ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `entity_command` (in Nightly builds, run with -Z macro-backtrace for more info)
